gradle.properties

```diff
@@ -27,3 +27,6 @@ systemProp.org.gradle.kotlin.dsl.precompiled.accessors.strict=true
 
 # Default performance baseline
 defaultPerformanceBaselines=7.6-commit-0f1793e
+
+enableConfigurationCacheForDocsTests=true
```

Comment:
If this is enabled by default, does this mean we lose some docs test coverage with CC disabled?

Reply:
Thank you for the pointer. That's accidental change, I didn't mean to commit this.

Reply:
If not enabled by default, how can we test it on CI?

Reply:
> Do you want to add a new build configuration to test `configCacheDocsTest`?

Yes, that's my next planned step: either add a separate config or add this configuration alongside the existing `docsTest` run. Though I'm going to do that in the next PR.